src/app/meditation/chart/meditation-chart.css

```diff
@@ -0,0 +1,11 @@
+.chart {
+  display: inline-block;
+  height: 140px !important;
+  width: 100%;
```

Comment:
Would it be possible to add a small `margin-top` (~5px) to the chart? I think it looks better if it's not touching the tab bar above.